Packs/GLIMPS_Detect/Integrations/GLIMPSDetect/GLIMPSDetect.yml

```
@@ -0,0 +1,261 @@
+commonfields:
+  id: GLIMPS_Detect
+  version: -1
+category: Forensics & Malware Analysis
+name: GLIMPS_Detect
+display: GLIMPS Detect
+fromversion: 6.0.0
+tests:
+- Run all tests
+description: Use the GLIMPS Detect Integration to send files to GLIMPS Malware and get results from it
+configuration:
+- display: API Token
+  name: api_token
+  type: 0
+  required: true
+- display: URL
+  name: url
+  type: 0
+  required: true
+- display: Trust any certificate (not secure)
+  name: insecure
+  defaultvalue: false
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: false
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- additionalinfo: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days, 3 months, 1 year)
+  defaultvalue: 7 days
+  display: First Fetch Time
+  name: first_fetch
+  required: false
+  type: 0
+- additionalinfo: Maximum number of alerts per fetch. Default and recommended is 50
+  defaultvalue: '50'
+  display: Fetch Limit
+  name: max_fetch
+  required: false
+  type: 0
+script:
+  feed: false
+  isfetch: true
```

```suggestion
  isfetch: false
```
There is no `fetch incidents` command. Do you intend on adding it? if not, please change